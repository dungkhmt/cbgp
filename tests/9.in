50 50
50 100
32 49
43 39
46 24
24 48
43 34
45 8
34 4
33 44
6 0
28 4
36 7
36 13
17 5
46 11
36 9
42 6
15 8
20 1
4 5
4 6
8 5
18 17
35 11
12 17
47 11
38 20
45 14
49 4
41 46
30 21
6 2
31 1
22 16
20 27
8 9
23 17
31 12
29 24
13 26
48 39
29 3
25 19
22 2
30 19
12 44
45 10
40 16
31 21
37 12
1 2
45 3
14 3
37 0
49 2
42 22
35 14
15 9
33 17
5 34
42 16
13 1
8 1
14 11
42 27
19 0
45 7
6 1
29 11
30 25
23 37
39 32
32 4
21 19
18 23
17 0
31 19
31 0
46 29
2 3
23 12
26 7
43 32
48 11
13 9
42 20
26 3
35 47
9 7
41 48
45 9
33 31
39 2
33 8
33 12
13 7
49 39
38 42
23 0
46 48
32 34
1
48 8 10 5 34 22 38 26 62 16 57 0 42 14 36 28 2 0 33 31 0 0 48 30 15 2 32 27 43 31 31 28 37 18 55 1 51 3 43 12 28 15 45 9 34 19 51 4 48 26 42 10 37 27 33 15 76 1 44 26 44 10 12 4 55 26 9 1 82 0 44 32 34 29 50 5 31 18 53 25 35 20 49 27 32 17 55 27 12 3 37 37 48 29 45 33 52 26 54 24 
