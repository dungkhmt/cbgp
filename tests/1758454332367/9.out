(32, 49), (43, 39), (46, 24), (24, 48), (43, 34), (45, 8), (34, 4), (33, 44), (6, 0), (28, 4), (36, 7), (36, 13), (17, 5), (46, 11), (36, 9), (42, 6), (15, 8), (20, 1), (4, 5), (4, 6), (8, 5), (18, 17), (35, 11), (12, 17), (47, 11), (38, 20), (45, 14), (49, 4), (41, 46), (30, 21), (6, 2), (31, 1), (22, 16), (20, 27), (8, 9), (23, 17), (31, 12), (29, 24), (13, 26), (48, 39), (29, 3), (25, 19), (22, 2), (30, 19), (12, 44), (45, 10), (40, 16), (31, 21), (37, 12), (1, 2), (45, 3), (14, 3), (37, 0), (49, 2), (42, 22), (35, 14), (15, 9), (33, 17), (5, 34), (42, 16), (13, 1), (8, 1), (14, 11), (42, 27), (19, 0), (45, 7), (6, 1), (29, 11), (30, 25), (23, 37), (39, 32), (32, 4), (21, 19), (18, 23), (17, 0), (31, 19), (31, 0), (46, 29), (2, 3), (23, 12), (26, 7), (43, 32), (48, 11), (13, 9), (42, 20), (26, 3), (35, 47), (9, 7), (41, 48), (45, 9), (33, 31), (39, 2), (33, 8), (33, 12), (13, 7), (49, 39), (38, 42), (23, 0), (46, 48), (32, 34), 
numberIntersection=0 
minDistance=1.0 
minAngle=0.07118940671001006 
sumAngle=46.86764650453222 
sumDistance=290364.485600632 
minEdgeLength=1.0 
sumEdgeLength=2982.228284243931 

0.0000000000 1.0000000000 0.0711894067 46.8676465045 290364.4856006320 1.0000000000 2982.2282842439
0: (160, 102),
1: (162, 24),
2: (101, 68),
3: (88, 74),
4: (106, 105),
5: (197, 111),
6: (130, 70),
7: (83, 70),
8: (194, 0),
9: (73, 64),
10: (0, 200),
11: (79, 95),
12: (188, 34),
13: (92, 65),
14: (74, 86),
15: (93, 63),
16: (124, 66),
17: (195, 109),
18: (184, 102),
19: (152, 87),
20: (133, 49),
21: (159, 89),
22: (124, 64),
23: (180, 101),
24: (89, 96),
25: (159, 85),
26: (87, 71),
27: (132, 56),
28: (147, 147),
29: (92, 90),
30: (158, 87),
31: (182, 33),
32: (99, 108),
33: (193, 2),
34: (108, 179),
35: (72, 88),
36: (83, 68),
37: (174, 96),
38: (129, 59),
39: (94, 103),
40: (123, 66),
41: (88, 97),
42: (128, 66),
43: (94, 109),
44: (189, 18),
45: (42, 67),
46: (85, 96),
47: (70, 90),
48: (93, 102),
49: (100, 103),
160 102 162 24 101 68 88 74 106 105 197 111 130 70 83 70 194 0 73 64 0 200 79 95 188 34 92 65 74 86 93 63 124 66 195 109 184 102 152 87 133 49 159 89 124 64 180 101 89 96 159 85 87 71 132 56 147 147 92 90 158 87 182 33 99 108 193 2 108 179 72 88 83 68 174 96 129 59 94 103 123 66 88 97 128 66 94 109 189 18 42 67 85 96 70 90 93 102 100 103 
0: (160, 102), 1: (162, 24), 2: (101, 68), 3: (88, 74), 4: (106, 105), 5: (197, 111), 6: (130, 70), 7: (83, 70), 8: (194, 0), 9: (73, 64), 10: (0, 200), 11: (79, 95), 12: (188, 34), 13: (92, 65), 14: (74, 86), 15: (93, 63), 16: (124, 66), 17: (195, 109), 18: (184, 102), 19: (152, 87), 20: (133, 49), 21: (159, 89), 22: (124, 64), 23: (180, 101), 24: (89, 96), 25: (159, 85), 26: (87, 71), 27: (132, 56), 28: (147, 147), 29: (92, 90), 30: (158, 87), 31: (182, 33), 32: (99, 108), 33: (193, 2), 34: (108, 179), 35: (72, 88), 36: (83, 68), 37: (174, 96), 38: (129, 59), 39: (94, 103), 40: (123, 66), 41: (88, 97), 42: (128, 66), 43: (94, 109), 44: (189, 18), 45: (42, 67), 46: (85, 96), 47: (70, 90), 48: (93, 102), 49: (100, 103), 
numberIntersection=0 minAngle=0.07118940671001006 sumAngle=46.867646504559595 minEdgeLength=1.0 sumEdgeLength=2982.228284243939 minDistance=1.0 sumDistance=290364.4856006386 
Time taken: 24.731 s
